library/core/src/num/int_macros.rs

```diff
@@ -2015,7 +2015,11 @@ macro_rules! int_impl {
         ///
         /// # Panics
         ///
-        /// This function will panic if `rhs` is 0 or the division results in overflow.
+        /// This function will panic if `rhs` is zero.
+        ///
+        /// ## Overflow behavior
+        ///
+        /// On overflow, this function will panic in debug mode and wrap in release mode.
```

Comment:
```suggestion
        /// On overflow, this function will panic if overflow checks are enabled (default in debug mode) and wrap if overflow checks are disabled (default in release mode).
```

(Needs word-wrapping. Should be applied to the other instances of this sentence as well.)

Reply:
Done.